README.md

```diff
@@ -214,6 +213,9 @@ Cluster autoscaling is supported for Windows instances.
 - Create a Windows node through a MachineSet (see spec in [Usage section](https://github.com/openshift/windows-machine-config-operator#usage)).
 - Define and deploy a [MachineAutoscaler](https://docs.openshift.com/container-platform/latest/machine_management/applying-autoscaling.html#configuring-machineautoscaler), referencing a Windows MachineSet.
 
+### Containerd Runtime
+Windows instances brought up with WMCO are set up with Containerd Container runtime.
```

Comment:
Consider including a link to `Containerd` main page.

Reply:
I am not suer if we are referring other submodules links like this. Ex contianernetworkruntime, kubeproxy and kubelet. Is there any specific reason you want to add containerd webpage link here?